src/components/ExampleBox.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
Why is this needed when we use 'allowSyntheticDefaultImports'?

Reply:
Not really sure, but I figured it was easier to change it than fighting some silly warning from the build system.